### ISSUE TITLE
If all attacking creatures have double strike, should the "hit" sound be played twice?

### ISSUE THREAD

**Issue by allentiak** [magarena/magarena#1402]
Or maybe when at least one attacking creature has double strike?

**Comment by melvinzhang** [comment]
Makes sense to play it when there is a creature that could deal damage in first strike/double strike damage set. Right now we always have this damage round which also doesn't follow the rules. 

This seems like a nice simple issue to work on, will likely involve src/magic/model/phase/MagicDefaultGameplay.java (to skip the First damage phase if there are no attacking/blocking creatures with first strike/double strike, and src/magic/model/phase/MagicCombatDamagePhase.java to remove the condition for playing the sound.

@allentiak would you like to give it a shot?

**Comment by allentiak** [comment]
Hi, Melvin,

Thanks a lot for considering me for this. Right now, however, I cannot give
it time :-(

I will keep it in mind for when I will have some slack.

Thanks again!
Leandro
